cmd/controller/main.go

```diff
@@ -40,6 +40,8 @@ import (
 
 	buildclientset "github.com/knative/build/pkg/client/clientset/versioned"
 	buildinformers "github.com/knative/build/pkg/client/informers/externalversions"
+	istioclientset "github.com/knative/pkg/client/clientset/versioned"
```

Comment:
One style question. I aliased the package `istioclientset` since those are the types we use, but if other types are vendored in the future, there could be other clients included in the same package. Should we rename `istioclientset` to `pkgclientset` (and every other similar name)?

Reply:
I think something like `sharedclientset`?  I think this is where common type definitions (e.g. KPA, Knative Ingress) might end up living.

Reply:
sounds good, I'll update

Reply:
renamed (almost) all the things